### DISCUSSION TITLE
Configuration for two different servers

### DISCUSSION THREAD

**Discussion by uroshsi** [ruyrybeyro/chrootvpn#28] [Q&A] [answered]
Hi,

first of all thank you for this script!

I have two clients, both of them are using Checkpoint for VPN access. How can I create and use two configurations?

Kind regards
Uros

**Comment by ruyrybeyro** [comment, marked as answer]
Hi,

Use one as is, with the default values:

 vpn.sh -i --vpn=FQDN_DNS_name_of_VPN

The 2nd one, call it via a script, giving it a different configuration file and chroot directory, as in:

 vpn.sh -i --vpn=FQDN_DNS_name_of_VPN2 --chroot /opt/chroot2 --file /opt/etc/vpn2.conf

After creating, for using the 2nd, you ought to add to every command --file /opt/etc/vpn2.conf 

Mind you both cannot be up at the  same time, as the VPN client uses the same service ports.

Best regards

**Reply by uroshsi** [reply]
Thank you!

Kind regards
Uros
